### DISCUSSION TITLE
[gatsby-theme-minimal-blog]: Using images with transparency

### DISCUSSION THREAD

**Discussion by ayhon** [LekoArts/gatsby-themes#642] [Questions] [answered]
Hello.
I'm setting up a small blog and in my about page I'd like to have an image with transparency. However, when I include the PNG in `content/pages/about/index.mdx` and visit ayhon.eu/about, the transparent part is replaced by a white color.

This is awfully apparent when using the dark mode.

![image](https://user-images.githubusercontent.com/43295942/130323981-dae6e8df-e972-4be0-b7e4-75868a4c3ed3.png)

Is this because of how the light/dark themes work, or am I doing something wrong?

**Comment by LekoArts** [comment, marked as answer]
Thanks for the note on this!
I'll publish a new minor to set a new default for everyone: https://github.com/LekoArts/gatsby-themes/pull/644

This should fix your issue

**Reply by LekoArts** [reply]
Please upgrade to the versions mentioned here and it should work: https://github.com/LekoArts/gatsby-themes/pull/645
